fv3core/stencils/updatedzd.py

```diff
@@ -233,6 +232,172 @@ def edge_profile_stencil(
         qe1x, qe2x, qe1y, qe2y = edge_profile_reverse(qe1x, qe2x, qe1y, qe2y, gam)
 
 
+class UpdateDeltaZOnDGrid:
+    """
+    Fortran name is updatedzd.
+    """
+
+    def __init__(self, grid, column_namelist, k_bounds):
+        self.grid = spec.grid
+        self._column_namelist = column_namelist
+        if any(
+            column_namelist["damp_vt"][kstart] <= 1e-5
+            for kstart in range(len(k_bounds))
+        ):
+            raise NotImplementedError("damp <= 1e-5 in column_cols is untested")
```

Comment:
this likely would be better suited in the future delnflux object, always separating these from where they were in the code could potentially cause some future confusion. But since we don't have the delnflux object and we don't want this check running every timestep, here is fine.

Reply:
That sounds like I'd agree. This would be done by having the column namelist be an initialization argument for delnflux instead of a call-time argument, so that this assertion can still happen on init. This probably is the same every timestep?